app.js

```diff
@@ -0,0 +1,34 @@
+const express = require("express");
+const cors =  require("cors");
+const app = express();
+const bodyParser = require("body-parser");
+const port = 3000;
+const school = require("./School.js")
+
+app.use(bodyParser.urlencoded({extended: false}))
+app.use(bodyParser.json())
+app.use(cors())
+
+app.get("/", (req, res) => {
+    res.send(school.classes)
+})
+app.post("/class/:className/addClass", (req, res) => {
+    res.send(req.body)
```

Comment:
in here you can use a method of your instance of your class `newSchool.addClass(name, teacher)`